app/workers/gamedev_worker.rb

```diff
@@ -1,19 +1,21 @@
 class GamedevWorker
-	include Sidekiq::Worker
+  include Sidekiq::Worker
 
-	def perform
-		response = RubyStackoverflow.questions({ order: 'asc', filter: 'withbody', pagesize: 100 })
-		response.data.each do | question|
-		  p =  ExternalPost.new(title: question.title,
-                            user_display_name: question.owner[:display_name],
-                            body_html: question.body,
-                            source_url: question.link,
-                            external_id: question.question_id)
-		  p.save
-	    end
-	end
-
-  Sidekiq::Cron::Job.create(name: "RubyStackoverflow fetch questions every friday",
-                            cron: "* * * * 5", klass: "GamedevWorker")
+  def perform
+    response = RubyStackoverflow.questions({ order: 'asc', filter: 'withbody', pagesize: 100 })
+    response.data.each do |question|
+      p = ExternalPost.new(
+        title: question.title,
+        user_display_name: question.owner[:display_name],
+        body_html: question.body,
+        source_url: question.link,
+        external_id: question.question_id)
+	    p.save
+	  end
```

Comment:
`end` at 14, 3 is not aligned with `response.data.each do |question|` at 6, 4<br>Tab detected.